applications/FluidDynamicsApplication/custom_strategies/strategies/residualbased_predictorcorrector_velocity_bossak_scheme_turbulent.h

```diff
@@ -285,8 +285,9 @@ namespace Kratos {
 
                     array_1d<double, 3 > & CurrentAcceleration = (itNode)->FastGetSolutionStepValue(ACCELERATION, 0);
                     array_1d<double, 3 > & OldAcceleration = (itNode)->FastGetSolutionStepValue(ACCELERATION, 1);
+                    array_1d<double, 3 > & RelaxedAcceleration = (itNode)->FastGetSolutionStepValue(RELAXED_ACCELERATION, 0);
```

Comment:
Does this have to be a historical variable?
Doesn’t seem so

Reply:
Time history of this is required to calculate adjoint sensitivities. At the moment, we are calculating it inside an HDF5 process and storing it in the nodes. This is to move the calculation outside the HDF5 application and to have a cleaner implementation.

Reply:
Ofc course you need the time history, I get that

But you are only accessing step “0“

I will talk to you tmr